fennel/internal_lib/to_proto/to_proto.py

```diff
@@ -1104,6 +1113,78 @@ def _renames_to_proto(renames: Optional[Dict[str, str]]) -> Mapping[str, str]:
     return renames
 
 
+def get_sampling_rate(sample: Optional[Union[float, Sample]]):
+    if isinstance(sample, float):
+        if sample < 0 or sample > 1:
+            raise ValueError("Sample rate should be between 0 and 1")
+        return sample
+    if isinstance(sample, Sample):
+        if sample.rate < 0 or sample.rate > 1:
+            raise ValueError("Sample rate should be between 0 and 1")
+        return sample.rate
+    raise ValueError(f"Invalid sample type: {type(sample)}")
+
+
+def get_sampling_columns(
+    sample: Optional[Union[float, Sample]],
+    dataset: Dataset,
+    preproc: Optional[Dict[str, PreProcValue]],
+):
+    disallowed_columns = [dataset.timestamp_field]
```

Comment:
lets make this empty given we have separate check for timestamp column below

Reply:
we are using this when user sent only sampling_rate as well.